File: login_and_signup_web-master/lib/Model/CrewChange.dart
class CrewChange {
  int crewChangeID;
  int custQuoteMasterID;
  int signOn;
  int signOff;
  String remarks;
  //int lastUpdatedBy;
  //String lastUpdatedDT;

  CrewChange({
    this.crewChangeID,
    this.custQuoteMasterID,
    this.signOn,
    this.signOff,
    this.remarks,
    //  this.lastUpdatedBy,
    //this.lastUpdatedDT,
  });

  factory CrewChange.fromJson(Map<String, dynamic> json) {
    return CrewChange(
      crewChangeID: json['crewChangeId'],
      custQuoteMasterID: json['custQuoteMasterId'],
      signOn: json['signOn'],
      signOff: json['signOff'],
      remarks: json['remarks'],
      //    lastUpdatedBy: json['lastUpdatedBy'],
      //  lastUpdatedDT: json['lastUpdatedDT'],
    );
  }
}

File: AniDAAPI/AniDAAPI/Models/CrewChange.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace AniDAAPI.Models
{
    public partial class CrewChange
    {
        public int CrewChangeId { get; set; }
        public int CustQuoteMasterId { get; set; }
        public int? SignOn { get; set; }
        public int? SignOff { get; set; }
        public string Remarks { get; set; }
        public int? LastUpdatedBy { get; set; }
        public DateTime? LastUpdatedDt { get; set; }
    }
}
